snow/engine/avalanche/transitive.go

```diff
@@ -312,9 +319,14 @@ func (t *Transitive) Chits(vdr ids.ShortID, requestID uint32, votes []ids.ID) er
 		response:  votes,
 	}
 	for _, vote := range votes {
-		if added, err := t.issueFromByID(vdr, vote); err != nil {
+		added, err := t.issueFromByID(vdr, vote)
```

Comment:
Can we change `added` to `issued` for consistency pls?

Reply:
done.